phpBB/phpbb/extension/base.php

```diff
@@ -137,6 +139,30 @@ protected function get_migration_file_list()
 
 		$migrations = $this->extension_finder->get_classes_from_files($migrations);
 
+		if ($validate_classes)
+		{
+			foreach ($migrations as $key => $migration)
+			{
+				// If the class exists and is a subclass of the
+				// \phpbb\db\migration\migration abstract class
+				// we skip it.
```

Comment:
This comment makes it sounds like you are going to skip classes that are valid subclasses.

How bout something like, 
> If a class does not exist or is not an extension of the \phpbb\db\migration\migration abstract class, unset it